Client/mods/deathmatch/logic/luadefs/CLuaPlayerDefs.cpp

```diff
@@ -40,6 +40,8 @@ void CLuaPlayerDefs::LoadFunctions()
         {"setPlayerNametagText", SetPlayerNametagText},
         {"setPlayerNametagColor", SetPlayerNametagColor},
         {"setPlayerNametagShowing", SetPlayerNametagShowing},
+        {"setPlayerHudComponentProperty", ArgumentParser<SetPlayerHudComponentProperty>},
+        {"resetPlayerHudComponentProperty", ArgumentParser<ResetPlayerHudComponentProperty>},
```

Comment:
It doesn't seem that these functions are related to the player. The word "Player" in a function name says that a player element is expected to be passed as one of the arguments. But none of your new function fit that.

Reply:
You're right, I just wanted to stay consistent with the existing HUD functions. However, the same could be said for other functions like ``getPlayerMapOpacity``, which also doesn't take a player argument but still includes 'player' in the name. Perhaps it's worth keeping the current naming to maintain consistency with other functions and leave the potential for someone to add this function server-side as shared in the future

Reply:
MTA Lua API is indeed has flaws and we already have examples of counterintuitive namings. Okay, let's leave it as it is.